passiveData/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/passivedata/ResultData.kt

```diff
@@ -0,0 +1,9 @@
+package org.sagebionetworks.assessmentmodel.passivedata
+
+import kotlinx.datetime.Instant
+
+interface ResultData {
```

Comment:
Doesn't this need to inherit from `org.sagebionetworks.assessmentmodel.Result` since that's the direction of your dependencies?